releasenotes.md

```diff
@@ -1,11 +1,22 @@
 # Release Notes
 
-## BOPTEST v0.5.0-dev
+## BOPTEST v0.6.0-dev
 
 Released on xx/xx/xxxx.
 
 **The following changes are backwards-compatible and do not significantly change benchmark results:**
 
+- Add the testing script and associated results of the baseline controllers for all the testcases. This is for [#495](https://github.com/ibpsa/project1-boptest/issues/495).
```

Comment:
Suggest "Add baseline testing scripts and associated KPI results for the baseline controllers of all the testcases."

Reply:
Changed